ocs_ci/ocs/resources/pod.py

```diff
@@ -375,17 +375,23 @@ def fillup_fs(self, size, fio_filename=None):
             self.io_params["filename"] = fio_filename
         self.fio_thread = self.wl_obj.run(**self.io_params)
 
-    def run_git_clone(self):
+    def run_git_clone(self, setup=False):
         """
         Execute git clone on a pod to simulate a Jenkins user
+
+        Args:
+            setup (bool): By default False, skips git package
```

Comment:
`setup` is fairly non-descriptive / generic. Can we call this `skip_install` or something that is more clear on what it does?

Reply:
Done